config/routes.rb

```diff
@@ -4,6 +4,7 @@
   resources :categories, only: [:show]
   resources :brands, only: [:show]
   resources :cart_items, only: [:create, :destroy, :update]
+  patch '/cart_quantity', to: 'cart_items#quantity'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.